storage/integration_test.go

```diff
@@ -2213,8 +2213,8 @@ func TestIntegration_ACL(t *testing.T) {
 		t.Errorf("default ACL: could not delete entity %s", entity)
 	}
 
-	entity2 := ACLEntity("user-jbd@google.com")
-	rule2 := ACLRule{Entity: entity2, Role: RoleReader, Email: "jbd@google.com"}
+	entity2 := ACLEntity("user-crwilcox@google.com")
```

Comment:
Is there another more stable test identity/email that is used for these tests elsewhere? Not to imply that you won't be around indefinitely 😄 

Also wondering if it's worth having a test with a specific email-- we might be able to just use another predefined role.

Reply:
I see https://github.com/googleapis/google-cloud-go/blob/master/storage/integration_test.go#L452 , let's do the same here?

Reply:
Hey now, I am gonna work at google and live *forever*. Obviously 😅 


https://github.com/googleapis/google-cloud-go/blob/master/storage/acl.go#L41 does define a few options, perhaps we can use a group instead?

Reply:
Could we use `project-owners-TEST_PROJECT_NUMBER` ==> `project-owners-123456789` instead? 

https://cloud.google.com/storage/docs/access-control/lists#scopes

Reply:
I set it to allAuthenticatedUsers, since that should be more-or-less permanent. are you implying though we could use an arbitrary product, or that we could use a known project?

Reply:
I would prefer specific to the test project which you should be able to get the project number: using `allAuthenticatedUsers` is almost like `allUsers` except `This special scope identifier represents anyone who is authenticated with a Google account`.

Reply:
As part of test setup we do require the following var

```
- `GCLOUD_TESTS_GOLANG_PROJECT_ID`: Developers Console project's ID (e.g.
bamboo-shift-455) for the general project.
```

I can make it use that as the principal.

Reply:
@tritone clarified that it's acceptable to keep allAuthenticatedUsers in this case. Please ignore my concern.